templates/upload.html

```diff
@@ -26,18 +26,20 @@ <h4 style='float: right;'> PDFs only</b> </h4>
 </form>
 
 {% if success %}
-<div id="message" class="alert alert-success text-center" style="width: 40%; margin: auto; margin-top: 5px;" role="alert"> Addition Successful! </div>
+<div id="message" class="alert btn-success text-center" style="width: 40%; margin: auto; margin-top: 5px;" role="alert"> Addition Successful! </div>
```

Comment:
I can't find the commit that changed these at a glance, but it feels to me like these alert->btn changes were a mistake.

Reply:
Yeah, these definitely need to be alert. `btn` makes them appear clickable.

Reply:
fixed